unittests/Analysis/CriticalPathTests.swift

```diff
@@ -0,0 +1,103 @@
+// This source file is part of the Swift.org open source project
```

Comment:
nit: is there a reason this code is using two-space tab sizes?

Reply:
Yeah, that's the project's default and I didn't change it after creating the file. I'll do now